nautobot_floor_plan/static/nautobot_floor_plan/css/svg.css

```
@@ -120,24 +129,23 @@ rect.object-orientation {
     text-transform: capitalize;
 }
 
-.highlighted {
-    stroke: #ff9900; /* Override stroke color */
-    stroke-width: 4px; /* Override stroke width */
-    stroke-opacity: 1; /* Full opacity for the stroke */
-    filter: drop-shadow(0 0 10px rgba(255, 153, 0, 0.9)); /* Add glow effect */
-    /* Note: We're not overriding fill or fill-opacity here */
+/* spotlight, border, and arrow styles */
+.spotlight-effect {
+    fill: rgba(255, 255, 0, 0.4);
```

Should this be the original value? os is this change intentional?
```suggestion
    fill: rgba(255, 255, 0, 0.8);
```

---

Setting back to original. I was playing around with it.